bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAdmin.java

```diff
@@ -159,11 +180,18 @@ public void createTable(TableDescriptor desc, byte[][] splitKeys) throws IOExcep
    * Calling {@link #addColumn(TableName, ColumnFamilyDescriptor)} was causing stackoverflow.
    * Copying the same code here. //TODO - need to find a better way
    */
+  // TODO(rahulkql): confirm if above case is still valid
   @Override
   public void addColumnFamily(TableName tableName, ColumnFamilyDescriptor columnFamilyDesc)
       throws IOException {
-    modifyColumns(tableName, columnFamilyDesc.getNameAsString(), "add",
-        ModifyTableBuilder.create().add(TableAdapter2x.toHColumnDescriptor(columnFamilyDesc)));
+    com.google.cloud.bigtable.admin.v2.models.ModifyColumnFamiliesRequest modifyColumnRequest =
```

Comment:
Please use `com.google.cloud.bigtable.admin.v2.models.ModifyColumnFamiliesRequest` as the import, and change all of the proto `ModifyColumnFamiliesRequest` uses to use the model as well.

Reply:
Updated imports for `models. ModifyColumnFamiliesRequest`